src/service/apiService.ts

```diff
@@ -0,0 +1,15 @@
+export const fetchData = (queryParam: string | undefined) => {
+  let url;
```

Comment:
I would suggest to use async/await syntax and try/catch block

Reply:
Not relevant